applications/tari_console_wallet/src/ui/app.rs

```diff
@@ -114,11 +114,8 @@ impl<B: Backend> App<B> {
     }
 
     pub async fn on_control_key(&mut self, c: char) {
-        match c {
-            'c' => {
-                self.should_quit = true;
-            },
-            _ => {},
+        if let 'c' = c {
```

Comment:
```suggestion
        if c == 'c' {
```

Reply:
does that work without a let? will try now